dashboard/modules/serve/serve_agent.py

```diff
@@ -110,11 +110,20 @@ async def delete_serve_application(self, req: Request) -> Response:
 
         return Response()
 
+    @routes.delete("/api/serve/applications")
+    @optional_utils.init_ray_and_catch_exceptions()
+    async def delete_serve_applications(self, req: Request) -> Response:
+        from ray import serve
```

Comment:
why do we need a delayed import here? if it's not necessary, move it. if it is, please add a comment

Reply:
I've followed the pattern in the file, and it seems this is the reason for the delayed imports for Ray Serve-related modules: https://github.com/ray-project/ray/blob/master/dashboard/modules/serve/serve_agent.py#L22